Makefile

```diff
@@ -1,10 +1,16 @@
-.PHONY: test build clean
+.PHONY: test build build-arwen clean
```

Comment:
`run-tests` should be added here as well. All `make` targets are phony, unless they are real filenames.

Reply:
Added.